conf/modules.config

```diff
@@ -19,11 +19,11 @@ process {
     ]
 
     withName: CUTN_TARGET {
-        ext.args = { "-n 10" }
+        ext.args = { "-n 10 -p 100000" }
```

Comment:
Are these values that could potentially be customisable by a user? If so might be worth adding as a parameter

Reply:
This is an arbitrarily big number to force `seqtk cutn` to be strict and not tolerate non-N sequences when detecting stretches of 10 or more Ns.  I do not expect users to change this value.